pkg/core/namespace.go

```diff
@@ -37,11 +39,11 @@ func (c *client) explicitOrConfigNamespace(namespaced Namespaced) string {
 
 func (kc *kubectlClient) NamespaceInit(options NamespaceInitOptions) error {
 
-	buildtemplateRelease := "https://storage.googleapis.com/riff-releases/riff-buildtemplate-0.1.0.yaml"
+	riffBuildRelease := "https://storage.googleapis.com/riff-releases/riff-build-0.1.1.yaml"
```

Comment:
where did this file come from? There is no such release in the riff-build repo, nor would the [publish script](https://github.com/projectriff/riff-build/blob/master/hack/publish.sh) create a release at this path.

Reply:
It is the curated and versioned artifact based on `riff-releases/latest/riff-build.yaml` which I think is produced by the CI.

Reply:
Let's use tagged releases rather than a random snapshot build. I'll cut a release.

Reply:
https://storage.googleapis.com/riff-releases/previous/riff-build/riff-build-0.1.0.yaml is available for your pleasure and is from the riff-build v0.1.0 tag.